doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -2369,6 +2425,13 @@ NCSDK-20967: The :ref:`nrf_rpc_entropy_nrf53` sample does not work on the networ
 Other samples
 =============
 
+.. rst-class:: v2-7-0
+
+NCSDK-27518: Logs from the PPR core in the CoreMark sample stop to display at some point
+  Logs from the PPR core in the CoreMark sample stop to display at some point during runtime execution and appear only after device reboot.
+
+  **Affected platforms:** nRF54H20 DK
+
```

Comment:
Let's remove this entry as we fixed this issue in the recently merged PR:

https://github.com/nrfconnect/sdk-nrf/pull/16118